roles/privacyidea_ha/tasks/install_packages.yml

```diff
@@ -5,3 +5,7 @@
     state: present
     update_cache: yes
 
+- name: Install prometheus_client in PrivacyIDEA
+  ansible.builtin.command:
+  cmd: sudo -H /opt/privacyidea/virtualenv/bin/pip install prometheus_client
+  become: true
```

Comment:
May be the ansible pip module could be more elegant here? I think it supports virtualenvs.